packages/camera/camera_android/android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.2'
+        classpath 'com.android.tools.build:gradle:7.3.0'
```

Comment:
Can you bump this version too or do you run into the same issues as https://github.com/flutter/packages/pull/4597?

Reply:
Trying it now

Reply:
I also tried upgrading **only** the plugin AGP version (not the example) and that didn't cause any issues, so it should be fine to add this upgrade to this pr